vm/devices/get/guest_emulation_transport/src/process_loop.rs

```diff
@@ -1853,7 +1854,10 @@ async fn request_igvm_attest(
     };
 
     let response_length = response.length as usize;
-    if response_length > ALLOCATED_SHARED_MEMORY_SIZE {
+    if response_length == 0xFFFFFFFF {
```

Comment:
let's define the value in https://github.com/microsoft/openvmm/blob/b8910f97c258ca39c8f2e535fbc241ea1e1be3ff/vm/devices/get/get_protocol/src/lib.rs#L586